coalib/settings/.coafile

```diff
@@ -0,0 +1,3 @@
+[cli]
+bears = PEP8Bear,PyUnusedCodeBear
```

Comment:
I'm not sure if we want to ship this here, or even as a part of the main coala install. maybe you should modify `coala-quickstart` instead. Why are you adding this file?